lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -143,11 +143,12 @@ class Simulator {
 
   /**
    * @param {Node} node
-   * @return {NodeTimingIntermediate}
+   * @return {Required<NodeTimingIntermediate>}
```

Comment:
yeah this can be improved, filed https://github.com/GoogleChrome/lighthouse/issues/11692 for myself

Reply:
thank you @patrickhulce!